torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -199,17 +199,14 @@ ScalarType numpy_dtype_to_aten(int dtype) {
     case NPY_BOOL: return kBool;
     default:
       // Workaround: MSVC does not support two switch cases that have the same value
-      if (dtype == NPY_LONGLONG || dtype == NPY_INT64) {
+      if (dtype == NPY_LONGLONG || dtype == NPY_INT64)
```

Comment:
I understand why you made this change, but please change it back.

Not including braces is a common source of bugs when additional lines are added.  And the `else` clause makes it clear that this case was considered and `break;` is the correct return.  As it is written now that is not clear.

Reply:
Thanks, I have pushed the fix.